controllers/contact-controller.js

```diff
@@ -232,9 +233,8 @@ function convertContactsToSearchResults(contacts) {
       const dateTime = contact.createdAt;
       const name = `${contact.rawJson.childInformation.name.firstName} ${contact.rawJson.childInformation.name.lastName}`;
       const customerNumber = contact.number;
-      const { callType } = contact.rawJson;
-      // TODO (Gian): retrieveTags must be done here instead than in flex plugin
-      const categories = 'TBD';
+      const { callType, caseInformation } = contact.rawJson;
+      const categories = retrieveCategories(caseInformation.categories);
```

Comment:
This feels like it's coupling our frontend and backend too tightly together.  The logic we use for the "Unspecified/Other" display is a presentation issue that the backend shouldn't care about.

I'd propose an alternative where we return the category and the subcategory and then leave the frontend responsible for figuring out how to show it.  So we return a pair of "Missing Children" and "Unspecified/Other" rather than "Unspecified/Other - Missing Children".  There are a few different ways to represent this in JSON.  The most obvious to me is categories as keys with an array of subcategories.  That is, 
```
{
  ...
  categories: {
    "Missing children": [ "Runaway", "Unspecified/Other"],
    "Violence": [ "Bullying" ]
}
```
I'm open to other formats that also have a distinction between the categories and subcategories.

This hits on a larger question we need to talk about soon.  I think we want to design our system with strong decoupling of our frontend and backend, with the backend providing an API that could be useful for multiple clients.  This is a bigger discussion we'll have to have that's not directly part of this PR.